sql/ts/src/skdb_util.ts

```diff
@@ -51,7 +51,7 @@ export class SKDBTable extends Array<Object> {
     return row[cols[0]];
   }
 
-  onlyRow(): Object {
+  onlyRow(): any {
```

Comment:
I don't claim to know what I'm talking about, but when I was looking into enabling eslint I ran into various advice to use `Record<string, unknown>` or just `unknown` instead of `any`. Just mentioning.

Reply:
Oh very nice, I didn't know about Record. I'll look in to changing to this. And this is a good enough reason to finally setup eslint too. Watch this space.

Reply:
https://github.com/SkipLabs/skfs/pull/731 might be helpful